pkgs/development/tools/uftrace/default.nix

```diff
@@ -1,14 +1,14 @@
-{stdenv, fetchFromGitHub }:
+{stdenv, fetchFromGitHub}:
 
 stdenv.mkDerivation rec {
   name = "uftrace-${version}";
-  version = "0.9";
+  version = "0.9.1";
 
   src = fetchFromGitHub {
     owner = "namhyung";
     repo = "uftrace";
-    rev = "f0fed0b24a9727ffed04673b62f66baad21a1f99";
-    sha256 = "0rn2xwd87qy5ihn5zq9pwq8cs1vfmcqqz0wl70wskkgp2ccsd9x8";
+    rev = "5766876f56b1bfcad76370d20f9f8b002a01e57d";
```

Comment:
Please use `rev = "v${version}"` here. Manually updating the rev every update is redundant and error-prone since [upstream properly tags their releases](https://github.com/namhyung/uftrace/releases) and `rev` understands tags.

Reply:
Good tip. Thanks, @timokau!